Update the HTML details element

mfuji09 opened the pull request:
* Fixed the description of the open attribute. If the "false" value is set on this attribute, this element opens.
* Added a new style which uses the details[open] selector in an example - https://github.com/mdn/content/issues/239

mfuji09 committed:
Update the HTML details element

* Fixed the description of the open attribute. If the "false" value is set on this attribute, this element opens.
* Added a new style which uses the details[open] selector - https://github.com/mdn/content/issues/239

ericwbailey commented on files/en-us/web/html/element/details/index.html:
```diff
@@ -82,7 +82,8 @@ <h2 id="Attributes">Attributes</h2>
 <dl>
  <dt>{{htmlattrdef("open")}}</dt>
- <dd>This Boolean attribute indicates whether or not the details — that is, the contents of the <code>&lt;details&gt;</code> element — are currently visible. The default, <code>false</code>, means the details are not visible.</dd>
+ <dd>This Boolean attribute indicates whether or not the details — that is, the contents of the <code>&lt;details&gt;</code> element — are currently visible. By default this attribute is absent which means the details are not visible.<br>
```
Just one question: What's the reasoning behind using a `br` in place of a paragraph? Seems like the note is a separate thought, so I'd think it'd be wrapped in `p` tags.

mfuji09 replied:
Good idea. It seems much beter. I try to do it.

mfuji09 committed:
Update the HTML details element

- divided the description of the open attribute into two paragraphs; an explanation of the attribute and a note.
- added "note" class on the note paragraph.

mfuji09 commented:
@ericwbailey I have corrected it, so please review again.

ericwbailey reviewed (approved):
Nice, thank you!
